package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "babel-eslint": "^7.1.1",
-    "eslint": "^3.14.1",
+    "eslint": "^3.16.1",
```

Comment:
Stupidly you need to keep the peerDependencies and devDependencies in sync, can you update the peerDependencies